src/java/test/org/apache/zookeeper/server/quorum/CommitProcessorConcurrencyTest.java

```diff
@@ -106,6 +106,7 @@ public void shutdown() {
 
                 @Override
                 public void notifyStopping(String threadName, int errorCode) {
+                    Assert.fail("Commit processor crashed " + errorCode);
```

Comment:
I think the two commits are different - they changed different part of code. It is indeed confusing because they share same commit message, which is better to be fixed via git amend, but in our case it is also fine to leave them as they are because when merge the pull request the committer will squash the commits into a single commit anyway.